<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify</class>
 <widget class="QDialog" name="modify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>367</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/修改用户信息.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>-50</y>
      <width>431</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <zorder>frame_2</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>153</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;原始密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>386</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;手机&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_edit">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>339</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>339</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;邮箱&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>293</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;用户名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirm_edit">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>246</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="original_key_edit">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>153</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;新密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>432</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;管理员密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_edit">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>293</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>246</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;确认新密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="admin_edit">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>432</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_but">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>479</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_edit">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>386</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="key_edit">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>200</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>500</y>
     <width>120</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/resource/22.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>431</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/resource/surrounding.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>490</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/resource/33.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>100</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;修改个人信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_but">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>500</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>frame_5</zorder>
  <zorder>label_8</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame_4</zorder>
  <zorder>admin_edit</zorder>
  <zorder>confirm_edit</zorder>
  <zorder>email_edit</zorder>
  <zorder>exit_but</zorder>
  <zorder>key_edit</zorder>
  <zorder>name_edit</zorder>
  <zorder>original_key_edit</zorder>
  <zorder>phone_edit</zorder>
  <zorder>submit_but</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
